profiles/urls.py

```diff
@@ -12,6 +13,8 @@
     path('saved-list/<pk>/', SavedCompaniesDestroy.as_view(), name='saved_companies_destroy'),
     path('viewed-list/', ViewedCompanyList.as_view(), name='viewed_company_list'),
     path('regions/', RegionListView.as_view(), name='region-list'),
-    path('category-list/', CategoryList.as_view(), name='category_list'),
-    path('activity-list/', ActivityList.as_view(), name='activity_list'),
+    path('category/', CategoryList.as_view(), name='category_list'),
+    path('activity/', ActivityList.as_view(), name='activity_list'),
+    path('category/<pk>', CategoryDetail.as_view(), name='category-detail'),
+    path('activity/<pk>', ActivityDetail.as_view(), name='activity-detail'),
```

Comment:
Previously we used plural forms of the collection name. And it's a common practice for REST APIs in general. Here's [an article](https://www.freecodecamp.org/news/rest-api-best-practices-rest-endpoint-design-examples/) on it. Please, change your urls accordingly.

Reply:
Thank you for feedback. But we have this naming of urls in our profiles: /profiles/ and profiles/<pk>. And Heorgii in last meeting told that we need to write some uniform code. So I tried to write similar to existing urls

Reply:
Thank you for feedback. Done